libs/models/src/lib/state/leaderboard.ts

```diff
@@ -0,0 +1,24 @@
+import { AppPlayerId } from './player';
```

Comment:
add consistent spacing between the types

Reply:
Add consistent spacing 755e6524b5cc463844463d50060be3604003a6a2